client/functions.lua

```diff
@@ -443,6 +443,8 @@ function QBCore.Functions.GetVehicleProperties(vehicle)
     if DoesEntityExist(vehicle) then
         local pearlescentColor, wheelColor = GetVehicleExtraColours(vehicle)
 
+        local colorPrimaryType, _, _ = GetVehicleModColor_1(vehicle)
+        local colorSecondaryType, _ = GetVehicleModColor_2(vehicle)
```

Comment:
You can remove these underscores, there are no other variables after it being uses so it won't cause issues